Functions.Templates/Templates/KafkaOutput-CSharp/KafkaOutputCSharp.cs

```diff
@@ -18,7 +18,7 @@ public class KafkaOutputCSharp
             [Kafka("BrokerList",
                    "topic",
                    Username = "$ConnectionString",
```

Comment:
Connection string will include all the information needed to connect right? do you still need username?

Reply:
Yes. it is the rule of the Kafka client connect to EventHubs.